scripts/west_commands/runners/intel_adsp.py

```diff
@@ -68,9 +73,15 @@ def do_add_parser(cls, parser):
                             help='the default basename of the key store in board.cmake')
         parser.add_argument('--key',
                             help='specify where the signing key is')
-        parser.add_argument('--pty', action="store_true",
-                            help='the log will not output immediately to STDOUT, you \
-                            can redirect it to a serial PTY')
+        parser.add_argument('--pty',
+                            help='the log will be output immediately to STDOUT, the \
+                            remote log host address is necessary.')
+        parser.add_argument('--run-opt', default=[], action='append',
+                            help='''Additional options for run/request service tool,
+                            e.g. \'--lock\' ''')
+        parser.add_argument('--log-opt', default=[], action='append',
+                            help='''Additional options for log service tool,
```

Comment:
```suggestion
                            help='''Additional options for `cavstool`
```
Or is it for `cavstool_server`?